armi/physics/neutronics/latticePhysics/latticePhysicsInterface.py

```diff
@@ -81,14 +81,7 @@ def __init__(self, r, cs):
 
         # Set to True by default, but should be disabled when perturbed cross sections are generated.
         self._updateBlockNeutronVelocities = True
-        # Geometry options available through the lattice physics interfaces
-        self._ZERO_DIMENSIONAL_GEOM = "0D"
-        self._ONE_DIMENSIONAL_GEOM = "1D"
-        self._TWO_DIMENSIONAL_GEOM = "2D"
-        self._SLAB_MODEL = " slab"
-        self._CYLINDER_MODEL = " cylinder"
-        self._HEX_MODEL = " hex"
-        self._burnupTolerance = self.cs[CONF_TOLERATE_BURNUP_CHANGE]
+        self._burnupTolerance = self.cs["tolerateBurnupChange"]
```

Comment:
Any reason to change the constant `CONF_TOLERATE_BURNUP_CHANGE` to a string literal?

Reply:
This was just a merge conflict resolution error on my part